climada/hazard/centroids/centr.py

```diff
@@ -1017,6 +1017,26 @@ def apply_point(df_exp):
                                  .map_partitions(apply_point, meta=Point)
                                  .compute(scheduler=scheduler))
 
+    @staticmethod
+    def _centroids_resolution(centroids):
+        """ Return resolution of the centroids in their units
+
+        Parameters:
+            centroids (Centroids): centroids instance
+
+        Returns:
+            float
+        """
+        if centroids.meta:
+            res_centr = abs(centroids.meta['transform'][4]), \
+                centroids.meta['transform'][0]
+        else:
+            res_centr = u_coord.get_resolution(centroids.lat, centroids.lon)
+        if abs(abs(res_centr[0]) - abs(res_centr[1])) > 1.0e-6:
+            LOGGER.warning('Centroids do not represent regular pixels %s.', str(res_centr))
+            return (res_centr[0] + res_centr[1])/2
```

Comment:
Why return this value? This is meaningless e.g. when the centroids are not even a grid.

Reply:
as mentioned above - WF hazard needs to be on a grid as we map the FIRMS data points onto it.
But as suggested, can move the method back to the WF module

Reply:
it's adjusted